community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -177,6 +182,15 @@ public void migrate( File storeDir, File migrationDir, MigrationProgressMonitor.
         // must be a proper translation happening while doing so.
     }
 
+    private void migrateWithDirectMigration( File storeDir, File migrationDir,
+            RecordFormats oldFormat, RecordFormats newFormat ) throws IOException
+    {
+        DirectRecordStoreMigrator migrator = new DirectRecordStoreMigrator( pageCache, fileSystem, config );
+        //Not interested in migrating MetaData store.
+        StoreType[] stores = ArrayUtil.filter( StoreType.values(), type -> type != StoreType.META_DATA );
```

Comment:
How about we implement ArrayUtil#filter using the above... it's certainly much more convenient that using that big chunk of code every time you want to filter an array